README.md

```diff
@@ -2,6 +2,8 @@
 
 <img align="right" width="180px" src="./logo/kwok.svg"/>
 
+How to pronounce [KWOK]? It's like "cock" but with a "w" in the middle.
```

Comment:
I think we can just say "KWOK is pronounced as `/kwɔk/`" :)

Reply:
I look at other projects do not seem to use the phonetic symbols, are similar to use the well-known words to spell

https://www.linux.org/threads/how-do-you-pronounce-sudo.19389/
https://www.nginx.com/about/the-nginx-name

I don't have a hard opinion

Reply:
If we use phonetic symbols, some people may still pronounce it as "K work".

Reply:
> If we use phonetic symbols, some people may still pronounce it as "K-work".

I asked my 6-yrs old she can pronounce `/kwɔk/` correctly lol... she said it's part of the phonics lesson. So I guess that won't be ambiguous :) https://www.youtube.com/watch?v=LOecBjycJMw

Reply:
Perfect, updated.